PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -200,6 +202,13 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
+
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event()); 
+                    std::uint32_t seed = runNum_uint + 4*evNum_uint;
+                    m_random_generator.seed(seed);
```

Comment:
What matters here is the value of the generated shift, which becomes independent of the event content if lumi and event numbers are repeated.
Indeed, addition of the lumi alone is not enough.